soccer/src/soccer/planning/planner_node.cpp

```diff
@@ -182,8 +182,7 @@ std::optional<RJ::Seconds> PlannerForRobot::get_time_left() const {
     // TODO(p-nayak): why does this say 3s even when the robot is on its point?
     // get the Traj out of the relevant [Trajectory, priority] tuple in
     // robot_trajectories_
-    const auto latest_trajs = robot_trajectories_->get();
-    const auto& [latest_traj, priority] = latest_trajs.at(robot_id_);
+    const auto& [latest_traj, priority] = robot_trajectories_->get(robot_id_);
```

Comment:
TODO after break: try removing this call entirely (to see if it's this get() or the get() below that's causing concurrency issues)

Reply:
it's not this one